lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -809,6 +809,16 @@ module Resources
         'op': 'replace',
         'path': '/name/0/family',
         'value': 'Potter'
+      },
+      {
+        'op': 'test',
+        'path': '/name/0/id',
+        'value': 'CI-12782404-0'
+      },
+      {
+        'op': 'replace',
+        'path': '/name/0/given',
+        'value': %w[Harry James]
```

Comment:
Let's do this as a normal array of string without %w.

Reply:
done https://github.com/cerner/fhir.cerner.com/commit/02691d03f3ee00e71c0073987ee66d73d6db6328